main_test.go

```diff
@@ -143,6 +216,52 @@ func waitForKubeconfigSecret(clusterName, namespace string) error {
 	}
 }
 
+func waitForNodesReady(clusterName, namespace string, expected int) error {
+	data, err := getChildKubeconfig(clusterName, namespace)
+	if err != nil {
+		return err
+	}
+	kubeconf, err := ioutil.TempFile("", "kubeconfig")
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	defer os.Remove(kubeconf.Name())
+	if err := ioutil.WriteFile(kubeconf.Name(), data, os.FileMode(0644)); err != nil {
+		return errors.WithStack(err)
+	}
+
+	readyRe := regexp.MustCompile(`\s+Ready`)
+	timeout := time.NewTimer(5 * time.Minute)
+	ticker := time.NewTicker(5 * time.Second)
+	for {
+		select {
+		case <-timeout.C:
+			return errors.New("timed out waiting for ready nodes")
+		case <-ticker.C:
+			cmd := exec.Command("kubectl",
```

Comment:
Doesn't need to be in this PR, but would you want to switch to using client-go instead of shelling out to kubectl?

Reply:
yeah, agreed. i'll file an issue